src/crud/useMediaAttach.ts

```diff
@@ -1,10 +1,10 @@
-import { useGlobal } from 'reactn';
-import { ISharedStrings, ActivityStates, MediaFile } from '../model';
-import { orbitErr } from '../utils';
+import {useGlobal} from 'reactn';
+import {ISharedStrings,  ActivityStates,  MediaFile} from '../model';
+import { orbitErr} from '../utils';
```

Comment:
The spaces after curly brackets are not consistent. Here they are removed and lines 5-7 they are retained?

Reply:
My ts and tsx files aren't being formatted correctly.  I've tried to figure out why but probably need to look at it together.  This was me attempting to fix what it did.